src/gradient.rs

```diff
@@ -29,13 +26,13 @@ impl Hash for ColorStop {
 }
 
 // Override PartialEq to use to_bits for the offset to match with the Hash impl
-impl cmp::PartialEq for ColorStop {
+impl PartialEq for ColorStop {
```

Comment:
Ah, these are already in the prelude. I wonder how these ended up with these prefices